bugsnag-android-performance/src/test/java/com/bugsnag/android/performance/SpanJsonTest.kt

```diff
@@ -183,4 +193,30 @@ class SpanJsonTest {
             json,
         )
     }
+
+    @Suppress("UNCHECKED_CAST")
+    @Test
+    fun testJsonEncodingWithWarning() {
```

Comment:
It would be good to have a test with an unexpected non-`null` attribute, such as a `Span`